packages/kbn-utility-types/BUILD.bazel

```diff
@@ -7,6 +7,7 @@ PKG_REQUIRE_NAME = "@kbn/utility-types"
 
 SOURCE_FILES = glob([
   "src/serializable/**",
+  "src/dot.ts",
   "src/index.ts"
```

Comment:
NIT (outside of the scope of the PR, but) can't we just `src/**/*` here?

Reply:
I'm happy to change this, but I'm not really sure what the consequences are. Mind if I leave it?

Reply:
Yea, no problem, it's a nit.